charts/refarch-templates/templates/serviceMonitor.yaml

```
@@ -0,0 +1,25 @@
+    - interval: 30s
+      path: /actuator/metrics
+      port: http
+      scheme: http
+  namespaceSelector:
+    matchNames:
+      - {{$.Release.namespace}}
+  selector:
+    matchLabels:
+      {{- include "getLabels" $data | trim | nindent 4 }}
+{{- end }}
+{{- end }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)